### DISCUSSION TITLE
Add Conditions

### DISCUSSION THREAD

**Discussion by phongXEM** [directus/directus#21067] [Feature Requests]
### Summary

I have a collection
<img width="825" alt="Screenshot at Jan 13 11-27-18" src="https://github.com/directus/directus/assets/141018948/02c059ab-ce3e-4420-907a-8e297d3654b9">
i want 
1. If payment_method=balance then balance_after must be non-null (and enabled)
2. If payment_method!=balance then balance_after must be null (and disabled)
I set in condition field **balance_after** 
<img width="796" alt="Screenshot at Jan 13 11-33-41" src="https://github.com/directus/directus/assets/141018948/88262f4c-3988-42d4-8efb-f432447b1f4c">
Is it possible to set value to this field balance_after when setting the condition?

### Motivation

Set value for field when setting the condition in collection.

### Detailed Design

add in to condition

### Requirements List

Set value for field when setting the condition in collection.

### Drawbacks

Drawbacks

### Alternatives

Alternatives

### Adoption Strategy

Adoption Strategy

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
